packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html

```diff
@@ -34,26 +34,6 @@
                     ></span>
                 </span>
             </div>
-            <small
-                class="help-block"
-                data-ng-if="ctrl.targetIsRelativeDomain()"
```

Comment:
Also clean the controller if the function is not use anywhere else

Reply:
removed switchTargetToAbsolute definition and usage in DNAME too